legend-engine-test-runner-mapping/src/test/java/org/finos/legend/engine/test/runner/mapping/TestMappingTestRunner.java

```diff
@@ -84,205 +91,11 @@ public void testAssertionFailure() throws IOException
                 objectMapper.readValue(testResult.getActual().get(), JsonNode.class));
     }
 
-
```

Comment:
All these migrated to other places, right?

Reply:
yep

Reply:
Resolved